client-libs/python/openpipe/test_async_client.py

```diff
@@ -23,11 +23,9 @@ async def test_async_content():
     )
 
     last_logged = last_logged_call(client)
+    assert last_logged.req_payload["messages"][0]["content"] == "count to 3"
```

Comment:
Update tests to no longer expect messages to be nested under model response on the last logged call.